tests/test_other.py

```diff
@@ -11409,3 +11409,31 @@ def test_gmtime_noleak(self):
     # Confirm that gmtime_r does not leak when called in isolation.
     self.emcc_args.append('-fsanitize=leak')
     self.do_other_test('test_gmtime_noleak.c')
+
+  # Test that using llvm-nm works when response files are in use, and inputs are linked using relative paths.
+  # llvm-nm has a quirk that it does not remove escape chars when printing out filenames.
```

Comment:
You can do `@with_env_modify` on the function to avoid the extra block/indentation here.